app/pages/adduser/controller.js

```diff
@@ -1,33 +1,35 @@
-// import axios from 'axios';
+// import axios from 'axios'
```

Comment:
let's remove this now. We shouldn't need this in this file anymore.

Reply:
this file has not been implemented yet so will do when it has